pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -36,13 +38,23 @@ func SingleClientDHCPv6Server(
 
 	log.Log.Info("Starting SingleClientDHCPv6Server")
 
+	iface, err := net.InterfaceByName(serverIfaceName)
+	if err != nil {
+		log.Log.Infof("DHCPv6 - couldn't get the server interface: %v", err)
+	}
+
 	handler := &DHCPv6Handler{
 		clientIP:      clientIP,
 		leaseDuration: infiniteLease,
-		serverIface:   serverIface,
+		serverIface:   iface,
+	}
+
+	conn, err := handler.createConnection()
+	if err != nil {
+		return fmt.Errorf("couldn't create connection for dhcpv6 server: %v", err)
 	}
 
-	s, err := server6.NewServer(serverIface, nil, handler.ServeDHCPv6)
+	s, err := server6.NewServer("", nil, handler.ServeDHCPv6, server6.WithConn(conn))
```

Comment:
Since I use a custom connection the `ifame` parameter has [no effect](https://github.com/kubevirt/kubevirt/blob/e8b888b08034f93182a26d6bda1f56aee9230f13/vendor/github.com/insomniacslk/dhcp/dhcpv6/server6/server.go#L123).

Reply:
done.